common/src/main/scala/gu/msnotifications/Responses.scala

```diff
@@ -65,7 +66,11 @@ object RegistrationResponse {
   }
 }
 case class RegistrationResponse(registration: WNSRegistrationId, tags: List[String], channelUri: String, expirationTime: DateTime) {
-  def toRegistrarResponse : RegistrarResponse = ???
+  def toRegistrarResponse : RegistrarResponse = RegistrarResponse(
+    deviceId = channelUri,
+    WindowsMobile,
+    userId = Tags(tags.toSet).findUserId.get
```

Comment:
Option.get is bad.  Perhaps toRegistrarResponse could return an Option[RegistrarResponse].  No need for a space between method name and :